modules/persistent-storage-csi-efs-metrics-procedure-cli.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+:_mod-docs-content-type: PROCEDURE
+[id="efs-metrics-procedure-cli_{context}"]
+= Enabling usage metrics using the CLI
+
+To enable Amazon Web Services (AWS) Elastic File Service (EFS) storage Container Storage Interface (CSI) usage metrics using the CLI:
+
+. Edit the `ClusterCSIDriver` object by running the following command:
++
+[source, terminal]
+----
+$ oc edit clustercsidriver efs.csi.aws.com
+----
+
+. Under `spec.aws.efsVolumeMetrics.state`, change the value to `RecursiveWalk`.
```

Comment:
That field will likely not exist, so instead of saying "change" we should probably say "set" or something in that line.